app/upload/text/textbox.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {ReactNode, useState} from 'react';
+import { View, SafeAreaView, TouchableWithoutFeedback, Keyboard} from 'react-native';
+import TopRow from '../../../components/upload/text/TopRow';
+import BottomRow from '../../../components/upload/text/BottomRow';
+import TextBox from '../../../components/upload/text/TextBox';
+import { supabase } from '../../../lib/supabase';
+
+export default function Page() {
+    const [text, setText] = useState<string>("");
+    const [viewPromptOpen, setViewPromptOpen] = useState<boolean>(false);
+    const currentDate = new Date();
+    const currentDateIso = `${currentDate.getFullYear()}-${currentDate.getMonth() + 1}-${currentDate.getDate()}T00:00:00`;
+
+    async function uploadText(){
+        if (text === "") return;
+        const { error } = await supabase
+            .from('TextResponses')
+            .insert([
+                { text: text }
+            ]);
+        if (error) {
+            console.log("Error uploading text: ", error);
```

Comment:
Do we want to handle the case of giving a user the error feedback message? 

I'm thinking we can create a little toast component so that when there is an error we could prompt them to try again  to show that there is an error?